requirements/base.txt

```diff
@@ -138,7 +140,7 @@ pyjwt[crypto]==2.8.0
     # via
     #   drf-jwt
     #   edx-drf-extensions
-pymongo==3.13.0
+pymongo==4.4.0
```

Comment:
Will hold off merging this because of https://github.com/openedx/tutor-contrib-aspects/issues/891